corpus/templates/components/navbar_small.html

```diff
@@ -11,7 +11,10 @@
         </div>
     </div>
     <div class="navbar-center">
-        <a href="{% url 'index' %}" class="btn btn-ghost normal-case text-xl">Corpus</a>
+        <a href="{% url 'index' %}" class="btn btn-ghost normal-case text-xl">
+            <img src="../static/img/ieee-nitk.png" alt="IEEE NITK Logo" class="dark-logo w-20 h-10">
+            <img src="../static/img/ieee-nitk-white.png" alt="IEEE NITK Logo" class="light-logo w-20 h-10">
```

Comment:
Remove the `w-20` reference. This doesn't maintain the original aspect ratio, and instead squeezes the image into a 2:1 aspect ratio which is not what we want.

Reply:
fixed